build/docker/Datadog.Trace.ClrProfiler.Native.sh

```diff
@@ -7,16 +7,17 @@ cd "$DIR/../.."
 
 PUBLISH_OUTPUT_NET2="$( pwd )/src/bin/managed-publish/netstandard2.0"
 PUBLISH_OUTPUT_NET31="$( pwd )/src/bin/managed-publish/netcoreapp3.1"
+BUILD_TYPE=${buildConfiguration:-Debug}
```

Comment:
Could check if other scripts are missing similar fallback?

Reply:
will do

Reply:
Other scripts use `publishTargetFramework` and `buildConfiguration` but they do not assign any default value.

Do you think it is worth to make something like:
```
buildConfiguration=${buildConfiguration:-Debug}
publishTargetFramework=${publishTargetFramework:-netcoreapp3.1}
```
in all these files?

It may be handy for me as I develop in a quasi-Linux environment 😄

Reply:
Yes, go for it.

Reply:
Addressed: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/77/commits/90c8350836813a27e1af6ef3c563c1baf63d5f9d